src/jit/lsraarm.cpp

```diff
@@ -727,6 +727,43 @@ void Lowering::TreeNodeInfoInit(GenTree* tree)
             info->internalIntCount = 1;
             break;
 
+        case GT_COPY:
+        case GT_PUTARG_REG:
+            // TODO-ARM-Cleanup : the logic was copied from general case,
+            // Only the exceptional case was added - when src count should be 2.
+            info->dstCount = tree->IsValue() ? 1 : 0;
+            if (kind & (GTK_CONST | GTK_LEAF))
```

Comment:
@CarolEidt I just tried that but there is no easy way for the tree node to get register count. The reason is kind of similar to https://github.com/dotnet/coreclr/pull/12360/files#r123158463, that is `gtRegNum` and `gtOtherReg` are invalidated from Lowering phase.